src/status_im/ui/components/bottom_sheet/subs.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im.ui.components.bottom-sheet.subs
+  (:require [re-frame.core :as re-frame]))
+
+(re-frame/reg-sub
+ :bottom-sheet
+ (fn [{:bottom-sheet/keys [show? view]}]
+   {:show? show?
```

Comment:
Minor: there was a tiny macro in open-bounty repo (https://github.com/status-im/open-bounty/blob/31c85bfd94ab01b47e6cdbdff79982c2f5bd65ad/src/clj/commiteth/util/util.clj#L19) for constructing these maps.

Reply:
or select-keys

Reply:
I'm not sure that I understand how `select-keys` will solve this. Don't want to introduce new a new macro here.